### PR TITLE
feat: 🎸 ListPickerItem initial commit

### PR DESCRIPTION
**Pull request by billzhou0223**

### CODE REVIEW

File: Sources/FioriSwiftUICore/Views/ListPickerItem+View.swift [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                } value: {
+                    value
+                }
+            }
+        )
+    }
+}
+
+public extension ListPickerItem {
+    init(
```
Comment (MarcoEidinger) [inline Sources/FioriSwiftUICore/Views/ListPickerItem+View.swift:39]: That's the preferred initializer, correct? If yes then this should declaration should be documented

File: Sources/FioriSwiftUICore/Views/ListPickerItem+View.swift [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    ) {
+        self.init(key: key, value: value)
+            
+        if let configuration = configuration {
+            destinationView = configuration.destinationView
+        }
+    }
+}
+
+public struct ListPickerItemConfiguration {
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply Sources/FioriSwiftUICore/Views/ListPickerItem+View.swift:52]: I have to agree with Hound ;)

**Review by MarcoEidinger — APPROVED** [review]
Great work @billzhou0223 . I suggest to add documentation for the preferred initializer and the `ListPickerItemConfiguration` struct

P.S.: I am still surprised that generated code was apparently formatted. I'll have a look today. For this PR lets not worry about it.

**Comment by billzhou0223** [comment]
> Great work @billzhou0223 . I suggest to add documentation for the preferred initializer and the `ListPickerItemConfiguration` struct
> 
> P.S.: I am still surprised that generated code was apparently formatted. I'll have a look today. For this PR lets not worry about it.

Added in latest commit.

**Pull request merged**